src/main/java/org/truffleruby/RubyLanguage.java

```diff
@@ -126,13 +126,13 @@ protected boolean patchContext(RubyContext context, Env newEnv) {
 
     @Override
     protected void finalizeContext(RubyContext context) {
-        Log.LOGGER.fine("finalizeContext()");
+        Log.LOGGER.fine("finalize context");
         context.finalizeContext();
     }
 
     @Override
     protected void disposeContext(RubyContext context) {
-        Log.LOGGER.fine("disposeContext()");
+        Log.LOGGER.fine("dispose context");
```

Comment:
There are specs looking for these messages, so they will fail.
I think it's better to not rename as `initializeContext()` is much clearer that it is the TruffleLanguage context initialization and not some general "context creation" message.

Reply:
I'll revert.